HealthyTests/Classes/Modules/CreateAccount/Mocks/CreateViewModelMock.swift

```diff
@@ -0,0 +1,28 @@
+@testable import Healthy
+
+final class CreateAccountViewModelMock: CreateAccountViewModelType {
+
+    // MARK: - Properties
+    var updateUsernameCalled = false
+    var updateEmailCalled = false
+    var updatePasswordCalled = false
+    var updateConfirmPasswordCalled = false
+
+    // MARK: - Methods
+    func updateUsername(_ username: String) {
+        updateUsernameCalled = true
+    }
+    func updateEmail(_ email: String) {
+        updateEmailCalled = true
+    }
+    func updatePassword(_ password: String) {
+        updatePasswordCalled = true
+    }
+    func updateConfirmPassword(_ confirmPassword: String) {
+        updateConfirmPasswordCalled = true
+    }
+    func configureButtonEnabled(onEnabled: @escaping (Bool) -> Void) {
+    }
+    func updateAcceptTermsAndConditions(_ accepted: Bool) {
+    }
+}
```

Comment:
```suggestion
final class CreateAccountViewModelMock: CreateAccountViewModelType {
    // MARK: - Methods
    private(set) var updateUsernameCallCount: Int = .zero
    func updateUsername(_ username: String) {
        updateUsernameCallCount += 1
    }
    
    private(set) var updateEmailCallCount: Int = .zero
    func updateEmail(_ email: String) {
        updateEmailCallCount += 1
    }
    
    private(set) var updatePasswordCallCount: Int = .zero
    func updatePassword(_ password: String) {
        updatePasswordCallCount += 1
    }
    
    private(set) var updateConfirmPasswordCallCount: Int = .zero
    func updateConfirmPassword(_ confirmPassword: String) {
        updateConfirmPasswordCallCount += 1
    }
    
    private(set) var configureButtonEnabledCallCount: Int = .zero
    func configureButtonEnabled(onEnabled: @escaping (Bool) -> Void) {
        configureButtonEnabledCallCount += 1
    }
    
    private(set) var updateAcceptTermsAndConditionsCallCount: Int = .zero
    func updateAcceptTermsAndConditions(_ accepted: Bool) {
        updateAcceptTermsAndConditionsCallCount += 1
    }
}
```

Call count validation with rquals to 1 is much better to avoid multiple calls.